helm/robusta/templates/runner-service-account.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: {{ .Release.Name }}-runner-cluster-role
   namespace : {{ .Release.Namespace }}
 rules:
+  {{- if .Values.runner.customClusterRole.enabled }}
```

Comment:
Can we control it with a single attribute? (without the `enabled`)
For example, initialized it: `customClusterRoles: []`
Then just check the array, like we do [here](https://github.com/robusta-dev/robusta/blob/f559e738bf0414e1e5a04b49184f6715379c3a20/helm/robusta/templates/runner.yaml#L70)

Reply:
Fixed